src/main/webapp/solarized-definitions.css

```diff
@@ -71,7 +71,7 @@
   /* --font-size-monospace */                                                                                           /* not a color */
   /* --font-size-sm */                                                                                                  /* not a color */
   /* --font-size-xs */                                                                                                  /* not a color */
-  /* --header-bg-classic */
+  --header-bg-classic: #000;
```

Comment:
this wasn't set before and was relying on what was set in core.

It's one of the colours used in preview so needs to be set otherwise the active theme will show it's colour there:
https://github.com/jenkinsci/theme-manager-plugin/blob/master/src/main/resources/lib/theme-manager/selector.jelly#L31-L37